### ISSUE TITLE
`rentEpoch` in `AccountHeader` should be a `bigint`

### ISSUE THREAD

**Issue by Juici** [metaplex-foundation/umi#99]
`rentEpoch` return by the RPC is a `u64` and should be `bigint` in `AccountHeader` in order to not lose precision.

https://github.com/metaplex-foundation/umi/blob/eda8af5136a4e642cf6b4a7cbb7405ae405a64a0/packages/umi/src/Account.ts#L16-L21

If we call `getAccountInfo` (on devnet) for the public key (`vines1vzrYbzLMRdu58ou5XTby4qAqVRLmqo36NKPTg`) in the example for https://docs.solana.com/api/http#getaccountinfo, then the account info returned is:

```json
{
  "data": ["", "base64"],
  "executable": false,
  "lamports": 88819743038050,
  "owner": "11111111111111111111111111111111",
  "rentEpoch": 18446744073709551615,
  "space": 0
}
```

Here we can see the value for the `rentEpoch` is `18446744073709551615`, however as a `number` we lose precision and get `18446744073709552000`.

**Comment by lorisleiva** [comment]
Hi 👋 Thanks for reporting this. That's something I've also noticed since working on the new web3.js haha. I'm happy to accept a PR change for this change with a `minor` changeset.

**Comment by HarishPrasannaV** [comment]
Hey, I'm a beginner and would like to work on this. Any context and advice are highly appreciated.

**Comment by Juici** [comment]
Fixed in #105.
